code/game/machinery/harvester.dm

```diff
@@ -150,6 +146,10 @@
 	if(default_deconstruction_crowbar(I))
 		return TRUE
 
+/obj/machinery/harvester/wrench_act(mob/living/user, obj/item/I)
+	if(default_change_direction_wrench(user, I))
+		return TRUE
```

Comment:
```suggestion
	if(default_change_direction_wrench(user, I))
		return TRUE
	return ..()
```
We should return parent behaviours typically if it hasn't been handled. If you explicitly don't want parent behaviours, then return FALSE to ensure that all return cases are explicitly covered. Even though byond returns null by default, if we are going to return a value then its good practice to always return